tests/app/views/test_services.py

```diff
@@ -863,12 +868,13 @@ def test_should_not_index_on_service_put_if_db_exception(self):
                 content_type='application/json')
 
             assert_equal(search_api_client.index.called, False)
-            db.session.commit = Mock(side_effect=c)
             db.session.rollback()
```

Comment:
I think we do but give it a try. We have only mocked out the commit so the deletes / updates are still going through. If we do not either roll back or commit I think that subsequent calls may break.